README.md

```diff
@@ -141,6 +141,7 @@ _Related: [Content Management Systems (CMS)](#content-management-systems-cms)_
 - [Horahora](https://github.com/horahoradev/horahora) - Video hosting website and video archival manager for Niconico, Bilibili, and Youtube. `MIT` `Go`
 - [Omeka S](https://omeka.org/s/) - Omeka S is a web publication system for universities, galleries, libraries, archives, and museums. It consists of a local network of independently curated exhibits sharing a collaboratively built pool of items, media, and their metadata. ([Source Code](https://github.com/omeka/omeka-s)) `GPL-3.0` `Nodejs`
 - [LiveStreamDVR](https://github.com/MrBrax/LiveStreamDVR) `⚠` - An automatic Twitch recorder capable of capturing live streams, chat messages and stream metadata. `MIT` `Python/Nodejs`
+- [Picsur](https://github.com/rubikscraft/Picsur) `⚠` - A simple imaging hosting playtfom that allows you to easily host, edit, and share images. `Typescript`
```

Comment:
Why the `⚠` symbol? This is only required if the software depends on a proprietary service outside the user's control. As far as I can see it does not use an external non-free service.

Reply:
Sorry about that for some reason I originally thought it was, but I changed that and everything else